python-frontend/typeshed_serializer/tests/test_typeshed_serializer.py

```diff
@@ -0,0 +1,53 @@
+import unittest
```

Comment:
I agree, I must say I didn't focus too much on tests yet as I wasn't sure exactly how and what to test. I had in mind to do that when setting up the analysis on Next and have the tests actually run on Cirrus (so that they are actually useful).
Would that be okay for you or do you prefer to have more extensive testing before merging in the feature branch?

Reply:
I don't think we need to setup analysis on Next, neither tests on Cirrus in the scope of this PR.
I would suggest instead to manually run some python testing tool to get the coverage to what we usually accept for PRs (i.e. >= 95%)
Is that ok for you?